tests-arquillian/pom.xml

```diff
@@ -314,8 +314,8 @@
                                     <outputDirectory>${JBOSS_HOME}/standalone/configuration</outputDirectory>
                                     <resources>
                                         <resource>
-                                            <directory>${JBOSS_HOME}/docs/examples/configs</directory>
-                                            <includes>standalone-activemq-embedded.xml</includes>
+                                            <directory>${JBOSS_HOME}/standalone/configuration</directory>
```

Comment:
Same here, just delete the full execution.

Reply:
Done